ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -55,41 +31,87 @@ <h2 class="count-header" id="total-count">
         class="btn btn-primary"
         [disabled]="(isRequesting$ | async) || (total$ | async) === null || (total$ | async) === -1"
         (click)="saveModal.open()">Create Cohort</button>
+      <button
+        class="btn btn-link"
+        (click)="navigateTo('notebook')"
+        disabled
+        [popper]="'Export to notebook'"
+        [popperTrigger]="'hover'">
+        <clr-icon shape="export" class="is-solid" size="30"></clr-icon>
+      </button>
+      <button
+        class="btn btn-link"
+        (click)="deleteModal.open()"
+        [disabled]="(isRequesting$ | async) || !cohort"
+        [popper]="'Delete cohort'"
+        [popperTrigger]="'hover'">
+        <clr-icon shape="trash" class="is-solid" size="30"></clr-icon>
+      </button>
+      <button
+        class="btn btn-link"
+        (click)="navigateTo('review')"
+        [disabled]="(isRequesting$ | async) || !cohort"
+        [popper]="'Review participant level data'"
+        [popperTrigger]="'hover'">
+        <clr-icon shape="copy" class="is-solid disabled" size="30"></clr-icon>
+      </button>
     </div>
+    <h2 class="count-header" id="total-count">
+      Total Count:
+      <span *ngIf="(isRequesting$ | async)" class="spinner spinner-sm">
+        Loading...
+      </span>
+      <span class="count-header" *ngIf="!(isRequesting$ | async) && (total$ | async) !== -1">
+        {{ total$ | async | number }}
+      </span>
+      <span *ngIf="(error$ | async) && !temporal.flag">
+        <clr-icon
+          [popper]="errorPopper"
+          [popperTrigger]="'hover'"
+          shape="error-standard"
+          size="18"></clr-icon>
+      </span>
+      <span *ngIf="!(isRequesting$ | async) && ((temporal?.flag && temporal.tempLength && (total$ | async) === null) || (total$ | async) === -1)">
+        - -
```

Comment:
Just to confirm, this is intentional? (Fine if it is, just strange visually for me)

Reply:
The double `-` is intentional. I think Shanta may have added the space in between, which looks strange to me also, so I removed it.